conjur-oss/README.md

```diff
@@ -294,7 +294,8 @@ The following table lists the configurable parameters of the Conjur OSS chart an
 
 |Parameter|Description|Default|
 |---------|-----------|-------|
-|`account`|Name of the Conjur account to be used by the Kubernetes authenticator|`"default"`|
+|`account.name`|Name of the Conjur account to be used by the Kubernetes authenticator|`"default"`|
+|`account.create`|Controls wether or not a Conjur account is created automatically after installation|`false`|
```

Comment:
I'm wondering if we need a special section in the README to emphasize this option? I'll let you or others decide, since I tend to err on the side of being too chatty/liberal with adding sections.